src/birthder.js

```diff
@@ -5,10 +5,12 @@
 //   None
 //
 // Commands:
+//   begin group birthder
 //   hubot birthdays list - shows a list of users and their birthdays
 //   hubot birthday set <username> <date>.<month>.<year> - sets a birthday for the user (privileged: admins only)
 //   hubot birthdays on <date>.<month> - shows a list of users with a set birthday date
 //   hubot birthday delete <username> - deletes birthday for the user (privileged: admins only)
+//   end group
 //
```

Comment:
This changes doesn't relate to the PR theme.

Reply:
corrected